### ISSUE TITLE
Can't enable the toolbar

### ISSUE THREAD

**Issue by Saud-97** [stnkl/EverythingToolbar#240]
**Describe the bug**
I have installed EverythingToolbar-0.7.4.msi. and I have the latest Everything installed 

**Steps to reproduce**
First time installing the toolbar. Didn't work.

**Expected behavior**
To find the Everything toolbar inside the Toolbars context menu and enable it.

**Screenshots**
![image](https://user-images.githubusercontent.com/39028181/173194967-c1111ab9-cd77-4ca3-82cf-43226d22ed6a.png)
![image](https://user-images.githubusercontent.com/39028181/173194988-e530ca41-6299-4787-8ac4-67e73d2eb2ed.png)

**Log file output**
Add the log file output from `%TEMP%\EverythingToolbar.log` below.

**The file didn't exist in my machine.**

**Comment by stnkl** [comment]
Hi @Saud-97, after installing EverythingToolbar you have to open the Toolbars context menu twice for it to show up. Can you please confirm again that you did that without uninstalling inbetween?

**Comment by Saud-97** [comment]
Yes that was one of the first things I've checked. It didn't help. I tried reinstalling everything and reinstalling the toolbar.

**Comment by stnkl** [comment]
I can't really explain this. There was a similar issue a while back in #139.

> Is there another way to run the installer in cmd or to enable verbose mode? I can't see any errors or warnings at all.

You can check in the registry wether the entries have been set correctly. But this is everything the installer should be doing apart from copying files around. Does it look similar to this for you? 

![image](https://user-images.githubusercontent.com/17520641/108622974-9b761900-743c-11eb-9d42-10779a3e64d3.png)

**Comment by Saud-97** [comment]
I have checked the keys and it seems to be similar. I will check the issue you mentioned and see if any suggestions there worked for me.

![image](https://user-images.githubusercontent.com/39028181/173239724-948c27c3-ee84-4ae4-afd2-356986b01e2f.png)
![image](https://user-images.githubusercontent.com/39028181/173239725-53a7734a-7b45-48bf-90a3-0ceca0bc609a.png)

**Comment by Saud-97** [comment]
I tried what was suggested in the issue and downloaded an old beta version. Didn't work at all. Let me know if there is anything I need to do or check from my side.

**Comment by dr-baba** [comment]
I had this problem last year. The only fix I found was reinstalling windows unfortunately (installed a fresh Windows 11 when that came out).

**Comment by stnkl** [comment]
> I tried what was suggested in the issue and downloaded an old beta version. Didn't work at all. Let me know if there is anything I need to do or check from my side.

Thanks for trying! Unfortunately I really don't know what could be causing this. It looks like everything is in place the way it should be. 

We'll have to wait until more information is available to be able to fix this issue.

**Comment by Saud-97** [comment]
I have tried the manual installer and there was no errors during installation. But it still doesn't show in toolbars context menu :(.
https://github.com/stnkl/EverythingToolbar/wiki/Installation-per-user-(experimental)
```
Installing EverythingToolbar 0.7.4.0 to C:\Users\saud\Downloads\Programs\x\EverythingToolbar\...
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
The operation completed successfully.
Please open the taskbar context menu twice for EverythingToolbar to show up!
Press any key to continue . . .



```

**Comment by stnkl** [comment]
@Saud-97 Thanks again for trying those things. Unfortunately I am still clueless as to why all of this is happening.

**Comment by Saud-97** [comment]
If there were other ways to install or enable this toolbar by default for example let me know. I will gladly try it out.

**Comment by Dhyfer1** [comment]
![image](https://user-images.githubusercontent.com/39028181/175112280-ef088ded-904e-4ed5-abf2-1f14a32f0c4e.png)
 
Hi @Saud-97 Can you tell me how you customized your registry to show blue folders?

**Comment by Saud-97** [comment]
Can't remember I think there was a registry key for setting default folder icon.

**Comment by coder66** [comment]
I had the same issue. The Toolbar is not linked in the start menu. So you need to open Windows Explorer and navigate to "Programs" and then "EverythingToolbar". In that folder you will find the executable file. Clicking on that starts the 3-step assistant.

I am sure this is really frustrating for non tech users - please add a start menu entry for the toolbar app.

**Comment by Saud-97** [comment]
Can you share the file path and executable name?  I'm pretty sure I tried many things but couldn't enable it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by coder66** [comment]
Unfortunately I have a German OS, but you should be able to find it.

![Snap3](https://github.com/srwi/EverythingToolbar/assets/14278231/68f8c4e0-fdee-47bb-9e0a-3e532a1cfd28)

**Comment by srwi** [comment]
EverythingToolbar can be used in two different ways: Taskbar deskband (aimed for Windows 10 users) or Taskbar shortcut launcher (for Windows 11 users). This bug ticket is about the deskband not being visible in the taskbar context menu in Windows 10 (see original post).

Both of you @coder66 and @Saud-97 seem to be affected by the deskband problem so for both of you the workaround of using the Windows 11 launcher instead works. Normally in Windows 11 the launcher should be started automatically after setup, so there is no usability issue here, since Windows 10 users should not normally have to use it.

Unfortunately it seems to be quite random when this bug occurs and I am clueless why this could happen.

**Comment by coder66** [comment]
Thanks for the explanation. I am using Win10 (19045) and I had already tried the toolbar 1,5 months ago. I had the same result, I was not able to start the toolbar app. Now with 1.1 I decided to try again and the same result, the toolbar was nowhere to be seen.

So I thought, why not look for the executable. And I found it, started it and then all was fine. After I ran the assistant, it also showed in the taskbar context menu.

Maybe adding it to the start menu is an idea? Then at least people can easily find it and start it manually.

**Comment by srwi** [comment]
> So I thought, why not look for the executable. And I found it, started it and then all was fine. After I ran the assistant, it also showed in the taskbar context menu.

Are you saying that the deskband entry showed up in the "Toolbars" menu after starting and setting up the Launcher application or just as a regular taskbar icon?

![image](https://github.com/srwi/EverythingToolbar/assets/17520641/c3ac7834-caef-46e4-884b-ed75477b2aa7)

**Comment by srwi** [comment]
@Saud-97 Could you please try launching and setting up the EverythingToolbar.Launcher.exe and afterwards check if the toolbar entry is visible for you as well?

This is very surprising to me, as both of those methods *should* not affect each other. But I'll take any hint I can get.

**Comment by Saud-97** [comment]
I'm currently traveling but I will try this tomorrow when I get back.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Saud-97** [comment]
I think my input is no longer relevant because I upgraded to Windows 11 which based on the answers here it has no issues. When I had the issue I was on Windows 10. I honestly no longer need the toolbar because I only needed it when my windows 10 search was broken for some reason.
